src/plugins/intel_gpu/src/graph/impls/ocl/register.hpp

```diff
@@ -27,6 +27,7 @@
 #include "intel_gpu/primitives/experimental_detectron_prior_grid_generator.hpp"
 #include "intel_gpu/primitives/experimental_detectron_roi_feature_extractor.hpp"
 #include "intel_gpu/primitives/experimental_detectron_topk_rois.hpp"
+#include "intel_gpu/primitives/eye.hpp"
```

Comment:
duplicated includes?

Reply:
@OlehKravchyshyn Can you handle this in other open PR? This is already merged in master.